### ISSUE TITLE
Screen reader does not announce the name and role for 'Filter' controls in windows.

### ISSUE THREAD

**Issue by msftedad** [jqwidgets/jQWidgets#763]
**Describe the bug**
Screen reader does not announce the name and role for 'Filter' controls in windows. Screen reader announces as 'Clickable'. Screen reader users cannot understand what the Filter controls are or how to interact with them because their name and role are not announced. This leads to confusion and prevents users from applying or adjusting filters independently.

**To Reproduce**

1. Open the URL: https://www.jqwidgets.com/jquery-widgets-demo/demos/jqxgrid/filtering.htm?light in edge browser in windows.
2. Press down arrow key to navigate to the Filter controls in browse mode.
3. Verify whether screen reader announces the name and role for 'Filter' controls in windows or not.

**Expected behavior**
Screen reader should announce the name and role for 'Filter' controls in windows.
Screen reader should announce as 'Filter Button'

**Screenshots**

https://github.com/user-attachments/assets/7c8fa74b-ecd4-49bf-bd49-993259989e18
<img width="969" height="518" alt="Image" src="https://github.com/user-attachments/assets/a1091c11-bf78-451b-b131-2582ab2019bd" />

**Desktop (please complete the following information):**
OS: Windows 11 version 25H2 (OS Build 26220.7523)
Browser: Edge (Version 144.0.3719.35)
Screen Reader: NVDA (2025.3.2)

**Smartphone (please complete the following information):**
NA
**Additional context**

**Comment by bmarkov** [comment]
planned for the next release

**Comment by msftedad** [comment]
@bmarkov, Thanks for the update. Could you please let us know the expected release date or the target release version for this fix? This will help us plan our validation and track the issue accordingly.
